src/README.md

```diff
@@ -0,0 +1,20 @@
+# tesstrain.py
+
+Utilities for working with Tesseract 4 using artificial training data.
+
+## Install
+
+This package requires the Tesseract training tools to be available on your system.
+
+To install the PIP package, either use `pip install tesstrain` or `python setup.py install`. At least Python 3.6 is required.
```

Comment:
```suggestion
To install the PIP package, either use `pip install tesstrain` or `python setup.py install`. At least Python 3.7 is required.
```

Reply:
Python 3.6 is an old version which is no longer supported, so there is no need to support it here.

Reply:
You are right about this, although the code should run fine on Python 3.6 for now and therefore it should not hurt. To be honest, I actually used Python 3.6 for the migration as this our company did not yet migrate from it.